botocore/client.py

```diff
@@ -522,7 +530,7 @@ def _service_model(self):
         return self.meta.service_model
 
     def _make_api_call(self, operation_name, api_params):
-        request_context = {}
+        request_context = {'client_region': self.meta.region_name}
```

Comment:
I don't have a strong preference.  I suggested camel case because I prefer not to change unrelated code, but it should be a safe change to make either way (I think).